.infrastructure/11_certs.tf

```diff
@@ -20,3 +20,19 @@ resource "aws_acm_certificate" "auth" {
 
   provider = aws.us-east-1
 }
+
+## Certificate HTTPS for CMS Strapi
+module "acm" {
```

Comment:
```suggestion
module "cms_ssl_certificate" {
```

Reply:
I'd rename to something different, so that if you need to use it in a different resource it's better to see `module.cms_ssl_certificate` than `module.acm`, because IMHO if I see `module.acm` it's not clear what is resource represents that module